### DISCUSSION TITLE
DEVC 'INSIDE WALL TEMPERATURE' goes down unexpectedly.

### DISCUSSION THREAD

**Discussion by Otabek-san** [firemodels/fds#13053] [Q&A]
Dear FDS-SMV Community,

I am trying to make a simulation to identify the flame spreading mechanism over a clathrate hydrate.
Currently, I am facing some kind of unrealistic data change inside the hydrate temperature which goes downward.
I am doing 1-D heat transfer model, with hydrate in the center and surrounding adiabatic wall. In the middle is igniter.
I used &VENT namelist to represent a boundary condition. Boundary of wall temperature is also remains the same at lower temperature
 am doing simulation on Linux, and the FDS version is: FDS-6.8.0
Attached is the input file and temperature graph for better understanding.

Thank you in advance.

![Temperature](https://github.com/firemodels/fds/assets/58806107/17f1f454-f9e3-4961-a942-6a4214cfcc04)

[hydrate.txt](https://github.com/user-attachments/files/15803756/hydrate.txt)

**Comment by drjfloyd** [comment]
Your inputs define an endothermic reaction that occurs at ambient temperature.

**Comment by Otabek-san** [comment]
Thank you for your answer.
Could you please clarify, which line or which block gives such reaction.

Thank you in advance.

**Reply by drjfloyd** [reply]
You only have one reaction defined in your input file, so it would be that reaction.

**Comment by Otabek-san** [comment]
Thank you for your reply, I have checked this reaction
&REAC FUEL='METHANE', AUTO_IGNITION_TEMPERATURE = 420. /
So, how do I define other exothermic reaction with it?

**Reply by drjfloyd** [reply]
The solid uses the single solid phase reaction defined in your input file. This reaction is endothermic, and the A and E values define a reaction where the peak reaction rate is at very low temperatures (look at the .out file)

&MATL ID                    ='Methane_hydrate'
      A(1)                  =1.90E19
      E(1)                  =8.42E4
      DENSITY               = 912.
      CONDUCTIVITY          = 0.49
      SPECIFIC_HEAT_RAMP    = 'c_ramp'
      N_REACTIONS           = 1
      MATL_ID               = 'WATER'
      NU_MATL               = 0.87
      SPEC_ID               = 'METHANE'
      NU_SPEC               = 0.13
      EMISSIVITY            = 0.65
      HEAT_OF_REACTION      = 454 /     should be modified
